### PR TITLE
METRON-410: mysql_server's MySQL install causes mutually assured dest…

### PR DESCRIPTION
**Pull request by dlyle65535**
…ruction when installed on the same machine as the Ambari Hive MySQL

Allowed the user to indicate their using an existing MySQL instance:

a) Refactored the required database and index settings to a separate property sheet using enhanced configs.
b) Made the MySQL start/stop logic from the add user, remove user and geo ip database load conditional.
c) Auto-populated the Storm UI URL and MySQL Host Address from the Add Service Wizard Master Install Selection page.
d) Cleaned up some lingering metainfo.xml items since I was in the file anyway to add the theme.

Tested on Docker cluster following procedure here: [Metron MPack](https://www.evernote.com/l/AhLFVR-9CsFIYYnOnF43BlxSsT4F856qwaY)

Test case 1: Default install (Ambari installed MySQL)l
Test case 2: Pre-existing MySQL install.

### CODE REVIEW

**Comment by dlyle65535** [comment]
Bump?

File: metron-deployment/packaging/ambari/metron-mpack/src/main/resources/common-services/METRON/0.2.1BETA/package/scripts/params/params_linux.py [hunk 1]
```diff
@@ -119,16 +119,21 @@
...
+
 mysql_port = config['configurations']['metron-env']['metron_enrichment_db_port']
 
 mysql_adduser_path = tmp_dir + "/addMysqlUser.sh"
 mysql_deluser_path = tmp_dir + "/removeMysqlUser.sh"
 mysql_create_geoip_path = tmp_dir + "/createMysqlGeoIp.sh"
 
-enrichment_hosts = default("/clusterHostInfo/enrichment_host", [])
-enrichment_host = enrichment_hosts[0] if len(enrichment_hosts) > 0 else None
+#enrichment_hosts = default("/clusterHostInfo/enrichment_host", [])
```
Comment (justinleet) [inline metron-deployment/packaging/ambari/metron-mpack/src/main/resources/common-services/METRON/0.2.1BETA/package/scripts/params/params_linux.py]: Doesn't seem like there's a reason to keep these comments around.  Can we just delete them?

Comment (dlyle65535) [inline metron-deployment/packaging/ambari/metron-mpack/src/main/resources/common-services/METRON/0.2.1BETA/package/scripts/params/params_linux.py]: No reason, I'll remove 'em- thanks!

**Comment by justinleet** [comment]
+1.  This is really great.

**Comment by dlyle65535** [comment]
Oh Travis. I just removed a comment.

**Pull request closed without merging**